### ISSUE TITLE
TravisCI flake8 failing on master branch (pydocstyle?)

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#1404]
This build on TravisCI https://travis-ci.org/biopython/biopython/builds/283656159 passed:

```
style installed: configparser==3.5.0,docutils==0.14,enum34==1.1.6,flake8==3.4.1,flake8-blind-except==0.1.1,flake8-docstrings==1.1.0,flake8-polyfill==1.0.1,flake8-rst-docstrings==0.0.7,mccabe==0.6.1,pycodestyle==2.3.1,pydocstyle==2.0.0,pyflakes==1.5.0,restructuredtext-lint==1.1.1,six==1.11.0,snowballstemmer==1.2.1
style runtests: PYTHONHASHSEED='789761084'
style runtests: commands[0] | flake8 BioSQL/
style runtests: commands[1] | flake8 Scripts/
style runtests: commands[2] | flake8 Doc/examples/
style runtests: commands[3] | flake8 Bio/
style runtests: commands[4] | flake8 Tests/
```

But https://travis-ci.org/biopython/biopython/builds/285670742 which was a cron triggered build for the same commit (so no code change on our part) failed:

```
style installed: configparser==3.5.0,docutils==0.14,enum34==1.1.6,flake8==3.4.1,flake8-blind-except==0.1.1,flake8-docstrings==1.1.0,flake8-polyfill==1.0.1,flake8-rst-docstrings==0.0.8,mccabe==0.6.1,pycodestyle==2.3.1,pydocstyle==2.1.0,pyflakes==1.5.0,restructuredtext-lint==1.1.1,six==1.11.0,snowballstemmer==1.2.1
style runtests: PYTHONHASHSEED='3202991917'
style runtests: commands[0] | flake8 BioSQL/
style runtests: commands[1] | flake8 Scripts/
Scripts/Restriction/ranacompiler.py:136:1: D107 Missing docstring in __init__
Scripts/Restriction/rebase_update.py:95:1: D107 Missing docstring in __init__
Scripts/Restriction/rebase_update.py:103:1: D107 Missing docstring in __init__
Scripts/debug/debug_blast_parser.py:44:1: D107 Missing docstring in __init__
Scripts/xbbtools/nextorf.py:35:1: D107 Missing docstring in __init__
Scripts/xbbtools/nextorf.py:55:1: D107 Missing docstring in __init__
Scripts/xbbtools/xbb_blast.py:42:1: D107 Missing docstring in __init__
Scripts/xbbtools/xbb_blastbg.py:33:1: D107 Missing docstring in __init__
Scripts/xbbtools/xbb_blastbg.py:120:1: D107 Missing docstring in __init__
Scripts/xbbtools/xbb_help.py:23:1: D107 Missing docstring in __init__
Scripts/xbbtools/xbb_search.py:33:1: D107 Missing docstring in __init__
Scripts/xbbtools/xbb_search.py:93:1: D107 Missing docstring in __init__
Scripts/xbbtools/xbb_translations.py:23:1: D107 Missing docstring in __init__
Scripts/xbbtools/xbb_utils.py:25:1: D107 Missing docstring in __init__
Scripts/xbbtools/xbb_widget.py:39:1: D107 Missing docstring in __init__
ERROR: InvocationError: '/home/travis/build/biopython/biopython/.tox/style/bin/flake8 Scripts/'
```

My guess is the version of pydocstyle is important, changing from ``pydocstyle==2.1.0`` to ``pydocstyle==2.0.0``

**Comment by peterjc** [comment]
Confirmed, they introduced D106 and D107 for what were previously special cases of D101 and D201.

**Comment by peterjc** [comment]
Closed via #1405
